web/themes/new_weather_theme/templates/block/block--weathergov-wfo-promo.html.twig

```diff
@@ -1,48 +1 @@
-<div class="grid-container padding-x-2">
-  <div class="grid-row">
-    <div class="grid-col tablet-lg:grid-offset-2 tablet-lg:grid-col-8">
-      <h2 class="text-normal text-primary-darker"> Your Local Forecast Office </h2>
-      <h3 class="margin-bottom-1">{{ content.name }}</h3>
-      <p class="line-height-sans-4 margin-top-0">
-        {{ content.shortIntro }}
-      </p>
-
-      {% if content.phone %}
-      <p>
-        <span class="font-family-mono font-mono-sm text-bold text-primary-darker text-uppercase"> {{ "Phone" | t }}: </span> <a href="tel:{{ content.phone }}" class="usa-link">{{ content.phone }}</a>
-      </p>
-      {% endif %}
-
-      {% if content.social %}
-      <p class="font-family-mono font-mono-sm text-bold text-primary-darker text-uppercase margin-bottom-1">
-        {{ "Find us on" | t }}
-      </p>
-        {% if content.social | length > 1 %}
-        <ul class="usa-list usa-list--unstyled"> 
-          {% for platform, link in content.social %}
-          <li class="li-margin-bottom-1">
-            <a href="{{ link }}" class="usa-link display-flex flex-align-center text-no-underline">
-              <svg class="usa-icon usa-icon--size-3 text-primary-darker margin-right-1 margin-top-neg-2px" aria-hidden="true" focusable="false" role="img">
-                <use xlink:href="{{ "/themes/new_weather_theme/assets/images/uswds/sprite.svg#" ~ platform}}"></use>
-              </svg>
-                <span class="">{{ platform | title | t }}</span>
-            </a>
-          </li>
-          {% endfor %}
-        </ul>
-        {% else %}
-        <p>
-          <a href="{{- content.social | first -}}" class="usa-link display-flex flex-align-center text-no-underline">
-            <svg class="usa-icon usa-icon--size-3 text-primary-darker margin-right-1 margin-top-neg-2px" aria-hidden="true" focusable="false" role="img">
-              <use xlink:href="{{ "/themes/new_weather_theme/assets/images/uswds/sprite.svg#" ~ content.social | keys | first }}"></use>
-            </svg>
-              <span class="">{{ content.social | keys | first | title | t }}</span>
-          </a>
-        </p>
-        {% endif %}
-
-      {% endif %}
-    </div>
-  </div>
-</div>
-
+{{ drupal_entity("node", content.node) }}
```

Comment:
Here's were we tell Twig to render the node. That'll pass us off to the `node--wfo-promo.html.twig` template.

Reply:
nice, this has turned out to be a sweet plugin